samples/linecounter/README.md

```diff
@@ -1,6 +1,16 @@
-# README
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-storage
+- azure-event-hubs
```

Comment:
Do we use KeyVault in here?

Reply:
Nope - https://github.com/Azure/azure-sdk-for-net/blob/master/samples/linecounter/LineCounter.csproj